### PR TITLE
numberingx:0.0.1

### PR DESCRIPTION
**Pull request by edhebi**
I am submitting
- [x] a new package
- [ ] an update for a package

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

**Description:** numberingx is an extension of the existing numbering patterns that follows the [css spec for counters](https://www.w3.org/TR/css-counter-styles-3/). It has support for a number of premade styles and allows users to define their own styles. While it's able to represent a number of exotic styles, its main use is localisation, as it allows user from any language to use the numbering style they see fit, without being limited to the ones in numbering().

### CODE REVIEW

File: packages/preview/numberingx-0.0.1/typst.toml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[package]
+name        = "numberingx"
+version     = "0.0.1"
+description = "Extended numbering patterns using the CSS Counter Styles spec"
+license     = "MIT-0"
+authors     = ["Edhebi"]
+repository  = "https://github.com/edhebi/numberingx"
+entrypoint  = "numberingx.typ"
+exclude     = [".git*"]
```
Comment (istudyatuni) [inline packages/preview/numberingx-0.0.1/typst.toml:9]: .git is ignored by default

Comment (edhebi) [reply packages/preview/numberingx-0.0.1/typst.toml:9]: that isn't the .git folder, it's for the gitattribute and .gitignore I have in the repo, but in the end I just didn't copy them over to the package registry to begin with.

File: packages/preview/numberingx-0.0.1/numberingx.typ [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: CC0-1.0
+
+#import "numberingx_impl.typ"
+
+#let format = numberingx_impl.format
+#let formatter = numberingx_impl.formatter
```
Comment (laurmaedje) [inline packages/preview/numberingx-0.0.1/numberingx.typ:6]:
```suggestion
#import "numberingx_impl.typ": format, formatter
```
that's simpler :)

File: packages/preview/numberingx-0.0.1/README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## Usage
+
+```ts
+// numberingx is expected to be imported with the syntax creating a named module
+#import "@preview/numberingx:0.0.1"
+
+// Use full-width roman numerals for titles, and lowercase ukrainian letters
+#set heading(numbering: numberingx.formatter(
+    "{fullwidth-upper-roman}.{fullwidth-lower-roman}.{lower-ukrainian}"
```
Comment (laurmaedje) [inline packages/preview/numberingx-0.0.1/README.md:14]: I would suggest using 2 spaces of indent, but it's up to you

**Pull request merged**